k8s/base/deploy-server.yaml

```diff
@@ -113,6 +113,34 @@ spec:
                 name: env
                 optional: false
 
+        - name: reconcile-filecache
```

Comment:
This init container runs the cache reconciliation on pod init. Unfortunately this means it will run once for each replica of this pod, but since it uses the filelock this should be safe, it will just delay initialization by a few extra seconds.